File: storage/storage-mysql/src/main/java/io/say/better/storage/mysql/dao/repository/SolutionSymbolWriteRepository.java
package io.say.better.storage.mysql.dao.repository;

import org.springframework.data.jpa.repository.JpaRepository;

import io.say.better.storage.mysql.domain.entity.SolutionSymbol;

public interface SolutionSymbolWriteRepository extends JpaRepository<SolutionSymbol, Long> {
}

File: storage/storage-mysql-kt/src/main/kotlin/io/say/better/storage/mysql/domains/solution/repository/SolutionSymbolWriteRepository.kt
package io.say.better.storage.mysql.domains.solution.repository

import io.say.better.storage.mysql.domains.solution.entity.SolutionSymbol
import org.springframework.data.jpa.repository.JpaRepository

interface SolutionSymbolWriteRepository : JpaRepository<SolutionSymbol, Long>
